io/net/virt-net/network_virtualization.py

```diff
@@ -210,19 +213,19 @@ def login(self, ipaddr, username, password):
         pxh.set_unique_prompt()
         pxh.prompt(timeout=60)
         self.pxssh = pxh
+        return pxh
 
-    def run_command(self, command, timeout=300):
+    def run_command(self, command, console, timeout=300):
```

Comment:
@smruti77 function readability is better if console obj is read first and command is read next. Just a suggestion.

`     def run_command(self, console, command, timeout=300): `

Reply:
ok